xwayland/selection/selection.c

```diff
@@ -51,12 +107,12 @@ bool xwm_selection_transfer_get_selection_property(
 	xcb_get_property_cookie_t cookie = xcb_get_property(
 		xwm->xcb_conn,
 		delete,
-		transfer->selection->window,
+		transfer->window,
```

Comment:
Doesn't this break outgoing?

Reply:
No, because this function is only used for incoming transfers. That's confusing though, so I've moved it to `incoming.c` and renamed it appropriately.